src/cpp/benders/benders_core/include/core/ILogger.h

```diff
@@ -55,6 +55,8 @@ struct LogData {
   double max_iterations;
 };
 
+const std::string indent_0 = "\t\t";
+const std::string indent_1 = "\t";
```

Comment:
I think that these constants should be where the logger are implemented. These are implementation details that belongs to the classes that actually print